helmfile-dependency-check/action.yml

```
@@ -0,0 +1,16 @@
+name: "helmfile-dependency-check"
+description: "Check for helmfile dependency updates"
+runs:
+  using: "node12"
+  main: "dist/index.js"
+inputs:
+  working-dir:
+    description: "The directory to run the action in"
+    default: "."
+outputs:
+  helmfile-lock-state:
+    description: "State of the helmfile lock. [missing, stale, fresh, update_available]"
```

```suggestion
    description: "State of the helmfile lock. [missing, fresh, update_available]"
```
Stale doesn't appear to be a valid value anymore